powerline/commands/main.py

```diff
@@ -107,6 +107,10 @@ def get_argparser(ArgumentParser=argparse.ArgumentParser):
 		'-w', '--width', type=int,
 		help='Maximum prompt with. Triggers truncation of some segments.'
 	)
+	parser.add_argument(
+		'--process-id', metavar='PID', type=int,
+		help='process_ID for the desired process'
```

Comment:
Documentation should start with a capital, end with dot, not use identifiers where they are not meant to (what’s wrong with “Process ID”?) and be more descriptive then “desired process”. I have no idea what “desired process” is supposed to mean.

Reply:
Would 'The process ID of process that renders powerline.' be okay for this?